linux_docker_resources/Dockerfile

```diff
@@ -8,17 +8,18 @@ ARG PLATFORM=x86
 # See: http://askubuntu.com/questions/506158/unable-to-initialize-frontend-dialog-when-using-ssh
 ENV DEBIAN_FRONTEND noninteractive
 
+RUN (apt-get update || true) && apt-get install -y locales
```

Comment:
Also if we have to do this, can we do it in a sed line like above for the `FROM ` so that it doesn't affect the normal linux jobs.